src/Components/Publishing/Article.tsx

```diff
@@ -110,72 +112,69 @@ export class Article extends React.Component<ArticleProps, ArticleState> {
   }
 
   renderFeatureArticle() {
-    const { headerHeight, isMobile } = this.props
+    const { headerHeight, isMobile, isSuper, relatedArticlesForCanvas } = this.props
     const { article } = this.state
 
+    const relatedArticleCanvas =
+      relatedArticlesForCanvas && !isSuper ? (
+        <RelatedArticlesCanvas articles={relatedArticlesForCanvas} vertical={article.vertical} />
+      ) : (
+        false
+      )
+
     return (
       <div>
-        <Header
-          article={article}
-          height={headerHeight}
-          isMobile={isMobile}
-        />
+        <Header article={article} height={headerHeight} isMobile={isMobile} />
 
         <FeatureLayout className="article-content">
-          <Sections
-            article={article}
-          />
+          <Sections article={article} />
         </FeatureLayout>
+
+        {relatedArticleCanvas}
       </div>
     )
   }
 
   renderStandardArticle() {
     const { isMobile, relatedArticlesForCanvas, relatedArticlesForPanel } = this.props
     const { article } = this.state
-    const relatedArticlePanel = relatedArticlesForPanel ?
-      <RelatedArticlesPanel
-        label={"Related Stories"}
-        articles={relatedArticlesForPanel}
-      />
-      : false
-
-    const relatedArticleCanvas = relatedArticlesForCanvas ?
-      <RelatedArticlesCanvas
-        articles={relatedArticlesForCanvas}
-        vertical={article.vertical}
-      />
-      : false
+    const relatedArticlePanel = relatedArticlesForPanel ? (
```

Comment:
Don't need the ternaries here -- can do (preferably in the render function so ones eyes don't have to skip around and things can be more 1:1)  
```javascript
{ relatedArticlesForPanel && 
  <RelatedArticlesPanel 
     label={"Related Stories"} 
     articles={relatedArticlesForPanel} 
   /> }

Reply:
Oo weird, this diff is from autosave, not new code. I'll change anyway, thanks!